bitbots_vision/src/bitbots_vision/vision_modules/yolo_handler.py

```diff
@@ -459,6 +462,38 @@ def predict(self):
                     self._candidates[class_name].append(c)
 
 
+class YoloHandlerPytorchYolo(YoloHandler):
+    """
+    Using Pytorch to get YOLO predictions
+    """
+    def __init__(self, config, model_path):
+        """
+        Initialization of PytorchYolo
+
+        :param config:
+        :param model_path:
+        """
+        weightpath = os.path.join(model_path, "yolo_weights.weights")
+        configpath = os.path.join(model_path, "config.cfg")
```

Comment:
These magic values should be moved somehow or be documented

Reply:
Do you want more documentation on what a yolo_weights.weights/config.cfg file is?